viscoelastic_rayleigh_taylor_drip/setup_plot.py

```diff
@@ -76,8 +78,9 @@
 ax.text(720,-30,'$Free$ $Surface$',fontsize=9)
 
 # Density subplot
-ax1.plot(density,zs,c='C0',linewidth=2)
-ax1.set_xlabel(r'$\rho$ $(kg/m^3)$',fontsize=13)
+cmap_batlow=cm.batlow #Batlow colors
+ax1.plot(density,zs,c=cmap_batlow(0.2),linewidth=2)
```

Comment:
Would it be possible to use `cm.devon_r` for density and `cm.bilbao` for viscosity as in the `vtk_plot.py` file? Maybe `cm.devonS_r` and `cm.bilbaoS` even works, that would be the categorical versions instead of the smooth sequential ones.